src/mongo/connectToMongoClient.ts

```diff
@@ -37,3 +42,8 @@ export class MongoConnectError extends Error {
         super(`Unable to connect to local Mongo DB instance. Make sure it is started correctly. See ${Links.LocalConnectionDebuggingTips} for tips.`);
     }
 }
+
+export function isLocalEmulatorConnectionString(connectionString: string): boolean {
+    const localEmulatorRegex: RegExp = new RegExp(`(localhost|127\.0\.0\.1):${encodeURIComponent(emulatorPassword)}`, 'i');
```

Comment:
What do you think - could we just check `return connectionString.includes(encodeURIComponent(emulatorPassword));`? Maybe check for the un-encoded password, too (is that possible?)

Seems virtually impossible that exact string would show up in other connection strings.

Reply:
Yeah that makes sense. It seems unlikely that `emulatorPassword` would be used elsewhere.

I tried attaching the emulator with the un-encoded password but the extension threw an error:

<img width="247" alt="Screen Shot 2020-07-09 at 12 33 21 PM" src="https://user-images.githubusercontent.com/22795803/87082802-659f5f80-c1e0-11ea-9b8d-02bdaa8a176f.png">

So I'll just verify the connection string includes the encoded password.